components/binary_sensor/index.rst

```diff
@@ -95,6 +95,9 @@ Supported filters:
   an OFF state. If an ON value is received while waiting, the OFF action is discarded. Or in other words:
   Only send an OFF value if the binary sensor has stayed OFF for at least the specified time period.
   **Useful for debouncing push buttons**.
+- **delayed_on_off**: Only send an ON or OFF value if the binary sensor has stayed in the same state
+  for at least the specified time period.
```

Comment:
LGTM, please just add this to the examples at the yaml block above.

Reply:
done